gh-pages/_docs/06-06-risk-profile.md

```diff
@@ -0,0 +1,14 @@
+---
+
+permalink: /docs/risk-profile/
+title: "Risk profile"
+
+The Risk Profile gives a quick overview about the complexity and risk structure of the code base. It tells you how much
+of the code is placed in files associated with low/high risk in means of maintainability.
+
+The separation of code in risk classes like low or high complexity is based on Java reference metric values and therefore
+can be inaccurate for other programming languages.
+
+To activate this function it is necessary to open the global configuration menu and activate the experimental features,
+after that the Risk profile button will appear in the main menu. Pressing this button will open the risk profile panel.
```

Comment:
Similar to the suspicious metrics, the hint about th e experimatal features is not neccessary. :)